strings.txt

```diff
@@ -8804,19 +8804,19 @@ SETUP_COMPOSERINARTISTS
 	SV	Spårets artist, Kompositör, band och orkester i Artister
 	ZH_CN	音轨艺人有关艺人的作曲家及乐队信息
 
-SETUP_COMPOSERINARTISTS_DESC
+SETUP_ROLESINARTISTS_DESC
 	CS	Informace o skladatelích, skupinách a orchestrech mohou být přidány do seznamu interpretů pro procházení a vyhledávání.
 	DA	Der kan vises oplysninger om hvem der har komponeret og hvem der spiller de forskellige numre, i listen med kunstnere når du gennemser eller søger i din musiksamling.
-	DE	Titelinformationen über Nummerets kunstner, Komponist, Gruppe und Orchester können in der Interpretenliste und beim Suchen aufgenommen werden.
-	EN	Song information about track artists, composers, bands and orchestras can be included in the artists list for browsing and search.
-	ES	La información de canción sobre artistas de pista, compositores, bandas y orquestas se puede incluir en la lista de artistas para examinar y buscar.
+	DE	Titelinformationen über Nummerets kunstner, Komponist, Gruppe und Orchester, usw, können in der Interpretenliste und beim Suchen aufgenommen werden.
```

Comment:
That's not German. Or not all of it 😁.

Reply:
Any suggestions?

God knows what we're really saying in languages other than English and German!

Reply:
That's why I usually (not always...) leave other languages alone. Or we ping previous translators when done? Please define "done" 🤦🏻 ...

Reply:
BTW: is "Song information about artists..." correct?

I believe the "Nummerets kunstner" in the German translation should be "Titelinterpret".